src/DynamoCoreWpf/Views/GuidedTour/HtmlPages/learnPackages.html

```diff
@@ -0,0 +1,45 @@
+        }
+
+        img {
+            display: block;
+            margin-right: auto;
+            width: 590px;
+            height: auto;
+        }
+
+        body {
+            font-family: "ArtifaktElement";
+            border: 0px;
+            font-size: 18px;
+        }
+
+        div {
+            margin:1px 23px 0px 25px
+        }
+    </style>
+</head>
+<body>
+    <img src="./Resources/DynamoPackages_1.png" />
+
+    <div>
+        A package is a toolkit of utilities that let you do more with
```

Comment:
I'm not sure if this text should be taken from a resource file or not, but I think localization could not be achieved in this way, @QilongTang should have a better idea about how localization works in WebPages. Thanks

Reply:
I have two ideas on this:

* We can set those texts with formatted strings that can be set by {0}{1} and then be replaced by a resource text.
* Set the Resource text by calling a javascript function